connectors/ssh/src/main/java/org/crsh/ssh/term/SSHLifeCycle.java

```diff
@@ -157,9 +157,9 @@ public void init() {
       server.setCompressionFactories(SSHFactories.setUpCompressionFactories());
 
       //
-      ArrayList<NamedFactory<Command>> namedFactoryList = new ArrayList<NamedFactory<Command>>(0);
+      ArrayList<SubsystemFactory> namedFactoryList = new ArrayList<>(0);
```

Comment:
The changes/type-casting here are a bit more mysterious to me - I don't know how valid it is. An intelli-J popup suggested this change to me and, given how type-casting seemed to work in SSHFactories.java, thought I'd try it here. Of course, everything built OK but not sure if my limited testing has gone anywhere near this code.